types/react-lazy-load-image-component/index.d.ts

```diff
@@ -1,13 +1,14 @@
-// Type definitions for react-lazy-load-image-component 1.3
+// Type definitions for react-lazy-load-image-component 1.5
 // Project: https://github.com/Aljullu/react-lazy-load-image-component#readme
 // Definitions by: Dan Vanderkam <https://github.com/danvk>
 //                 Diego Chavez <https://github.com/diegochavez>
 //                 Truong Hoang Dung <https://github.com/revskill10>
+//                 Kodai Suzuki <https://github.com/kodai3>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.5
 
 import {
-    CSSProperties,
+	CSSProperties,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-lazy-load-image-component/index.d.ts;
  Line: 11;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.